src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-            instance_logger.log("Received vertexCount: " + vertexCount, "info");
+        sockfd = socket(AF_INET, SOCK_STREAM, 0);
 
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string ownPartitions = (data);
-            ownPartitions = Utils::trim_copy(ownPartitions, " \f\n\r\t\v");
-            instance_logger.log("Received Own Partitions No: " + ownPartitions, "info");
-
-            /*Receive hosts' detail*/
-            write(connFd, JasmineGraphInstanceProtocol::SEND_HOSTS.c_str(),
-                  JasmineGraphInstanceProtocol::SEND_HOSTS.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::SEND_HOSTS, "info");
-
-            char dataBuffer[INSTANCE_LONG_DATA_LENGTH + 1];
-            bzero(dataBuffer, INSTANCE_LONG_DATA_LENGTH + 1);
-            read(connFd, dataBuffer, INSTANCE_LONG_DATA_LENGTH);
-            string hostList = (dataBuffer);
-            instance_logger.log("Received Hosts List: " + hostList, "info");
-
-            // Put all hosts to a map
-            std::map<std::string, JasmineGraphInstanceService::workerPartitions> graphPartitionedHosts;
-            std::vector<std::string> hosts = Utils::split(hostList, '|');
-            int count = 0;
-            int totalPartitions = 0;
-            for (std::vector<std::string>::iterator it = hosts.begin(); it != hosts.end(); ++it) {
-                if (count != 0) {
-                    std::vector<std::string> hostDetail = Utils::split(*it, ',');
-                    std::string hostName;
-                    int port;
-                    int dataport;
-                    std::vector<string> partitionIDs;
-                    for (std::vector<std::string>::iterator j = hostDetail.begin(); j != hostDetail.end(); ++j) {
-                        int index = std::distance(hostDetail.begin(), j);
-                        if (index == 0) {
-                            hostName = *j;
-                        } else if (index == 1) {
-                            port = stoi(*j);
-                        } else if (index == 2) {
-                            dataport = stoi(*j);
-                        } else {
-                            partitionIDs.push_back(*j);
-                            totalPartitions += 1;
-                        }
-                    }
-                    graphPartitionedHosts.insert(pair<string, JasmineGraphInstanceService::workerPartitions>(
-                        hostName, {port, dataport, partitionIDs}));
-                }
-                count++;
-            }
-            /*Receive file*/
-            write(connFd, JasmineGraphInstanceProtocol::SEND_FILE_NAME.c_str(),
-                  JasmineGraphInstanceProtocol::SEND_FILE_NAME.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::SEND_FILE_NAME, "info");
+        if (sockfd < 0) {
+            std::cout << "Cannot create socket" << std::endl;
```

Comment:
Lets replace the `cout` with a logger

Reply:
Removed all occurrences of `std::cout` in next PR

Reply:
Ok